src/Ditto.Sqs.Consumer/ConsumerOptions.cs

```diff
@@ -0,0 +1,21 @@
+using System;
+
+namespace Ditto.Sqs.Consumer
+{
+    public class ConsumerOptions
+    {
+        /// <summary>
+        /// The amount of delay to add to a sqs message if it cannot be handled
+        /// </summary>
+        public TimeSpan RetryDelay { get; set; } = new TimeSpan(0, 0, 15);
+
+        /// <summary>
+        /// The amount of times to retry a SQS message if it cannot be handled
+        /// </summary>
+        public int RetryAttempts { get; set; } = 5;
```

Comment:
I think, but this is an early guess, is that we will need to have some way to shovel (😂) stuff back from the deadletter queue if we are going to stick with this. EVS issue on Archive writes are probably going to persist longer than 5 * 15 seconds 🙈

Reply:
well... I wouldn't like to stick to this, but just think this is temporary measure and we are addressing the long term solution in the meantime. Coding a shoveler smells like that's is not the case